copilot/fsd-pre-award-application-deadline-reminders/manifest.yml

```diff
@@ -0,0 +1,69 @@
+name: fsd-pre-award-application-deadline-reminders
+type: Scheduled Job
+
+on:
+  schedule: "cron(30 9 * * ? *)"
+retries: 0
+timeout: 10m
+
+image: {}
```

Comment:
I guess this is just a placeholder? Maybe add a comment to indicate the `location` is injected by Github Actions?

Reply:
done 👍